### DISCUSSION TITLE
Update Description

### DISCUSSION THREAD

**Discussion by SchofieldPriest** [BlafKing/sd-civitai-browser-plus#102] [Ideas]
In a similar vein as the "Update assigned tags" feature under "Update Models", there should be an "Update description". More than a few models actually have prompts or important information (recommended weight, clip skip number, etc.) contained in the description, so having a feature that can rip the description of each installed model word for word could prove a great help.

**Comment by BlafKing** [comment]
Thanks for the suggestion!
This has now been implemented in v1.16, the "Update model tags" has been changed to "Update model info & tags".
The option now saves tags, description and base model version! :)

Additionally, there is now also an option to update/save model preview images!
